fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBInQueryTest.java

```diff
@@ -152,20 +168,20 @@ void testInQueryNoIndex() throws Exception {
         } else {
             assertMatchesExactly(plan,
                     predicatesFilterPlan(descendantPlans(scanPlan().where(scanComparisons(unbounded()))))
-                            .where(predicates(valuePredicate(ValueMatchers.fieldValueWithFieldNames("num_value_2"), new Comparisons.ListComparison(Comparisons.Type.IN, ImmutableList.of(0, 2))))));
+                            .where(predicates(valuePredicate(fieldValueWithFieldNames("num_value_2"), new Comparisons.ListComparison(Comparisons.Type.IN, ImmutableList.of(0, 2))))));
 
             assertEquals(738091077, plan.planHash(PlanHashable.CURRENT_LEGACY));
             assertEquals(-2062922437, plan.planHash(PlanHashable.CURRENT_FOR_CONTINUATION));
         }
         assertEquals(67, querySimpleRecordStore(NO_HOOK, plan, EvaluationContext::empty,
-                record -> assertThat(record.getNumValue2(), anyOf(is(0), is(2))),
+                rec -> assertThat(rec.getNumValue2(), anyOf(is(0), is(2))),
```

Comment:
While I was here, I've renamed `record` variables to `rec`. In JDK 17, record will be a keyword, though not strictly speaking a reserve word, so it will still be legal to have variables named `record`, though it is now frowned upon. Since I was already modifying a lot of this test class, I went ahead and cleaned this up, too